pom.xml

```diff
@@ -16,11 +16,11 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.11</maven.compiler.source>
+        <maven.compiler.target>1.11</maven.compiler.target>
 
-        <slf4j.version>1.7.25</slf4j.version>
```

Comment:
both slf4j and spring have good back compatibility. And old versions were used intended to avoid possible usage of new framework features. spring 4.3-compatible library will work fine in 4.3, 5.0, 5.1 and 5.2 spring (mostly), while 5.2.7 may lead to calls to classes that do not yet exist and will force the projects using this lib to upgrade (that may be a problem).